doc/user_guide/user_guide.md

```diff
@@ -354,6 +354,12 @@ By default PK configures the project to use Java 11. If you need to use a differ
 </properties>
 ```
 
+The specified Java version must be available on your local machine in [`toolchains.xml`](#configure-mavens-toolchainsxml). PK will automatically update the GitHub workflows `ci-build.yml` and `release.yml` and install the required Java versions using GitHub action [`setup-java`](https://github.com/actions/setup-java). This action automatically manages `toolchains.xml` on the GitHub runner, so that the build will use the same Java versions.
+
+When your project uses multiple Maven modules, each can specify it's own Java version. PK will collect all versions and use them in the GitHub workflows.
+
+PK will also calculate the maximum Java version used by all modules and use the next major Java version for generating `.github/workflows/ci-build-next-java.yml`.
```

Comment:
Aha, it's even a little bit different as my initial text proposal / my understanding in the conversation. :slightly_smiling_face: